tst/regression/test_suites/aniso_therm_cond_ring_conv/README.md

```diff
@@ -2,5 +2,5 @@
 
 Executes 2D ring diffusion problem following Sharma & Hammett (2007) and calculates convergence rate.
 Errors are calculated based on comparison to steady state solution.
-Convergence for this problem is not great, but matches other numbers reported in literate, e.g., Balsara, Tilley & Howk MNRAS (2008) doi:10.1111/j.1365-2966.2008.13085.x .
+Convergence for this problem is not great, but matches other numbers reported in literature, e.g., Balsara, Tilley & Howk MNRAS (2008) doi:10.1111/j.1365-2966.2008.13085.x .
```

Comment:
Do you have an image of the problem to share? Any obvious issues with AthenaPK's solution?